src/emulator/functionsEmulator.ts

```diff
@@ -551,7 +551,9 @@ export class FunctionsEmulator implements EmulatorInstance {
     for (const definition of toSetup) {
       // Skip function with invalid id.
       try {
-        functionIdsAreValid([definition]);
+        // Note - in the emulator, functionId = {region}-{functionName}, but in prod, functionId=functionName.
+        // To match prod behavior, only validate functionName
+        functionIdsAreValid([{ id: definition.name, platform: definition.platform }]);
```

Comment:
personal pref, but I'd do something like:

```
functionIdsAreValid([{ ...definition, id: definition.name }]);
```

Reply:
Ahhh, i do like this a bit more - done.